src/emitter.js

```diff
@@ -6,6 +6,16 @@ const {
   BLOCK,
 } = require("./encoding");
 
+// Takes an f64 on the stack and leaves an int32 boolean representing if it's
+// within epsilon of zero.
+const IS_ZEROISH = [op.f64_abs, op.f64_const, ...encodef64(0.0001), op.f64_lt];
```

Comment:
Awesome! I think its `0.00001` though: https://github.com/WACUP/vis_milk2/blob/master/ns-eel2/ns-eel-int.h#L56

Reply:
Wow. Thank goodness for code review!

Thanks! I'll fix before merging,.

Reply:
🙌